CONTRIBUTING.md

```diff
@@ -119,3 +119,15 @@ As with other OpenFeature SDKs, dotnet-sdk follows the
 This project includes a [`.editorconfig`](./.editorconfig) file which is
 supported by all the IDEs/editor mentioned above. It works with the IDE/editor
 only and does not affect the actual build of the project.
+
+## Benchmarking
+
+We use [BenchmarkDotNet](https://benchmarkdotnet.org/articles/overview.html) NuGet package to benchmark a code.
+
+To run pipelines locally, you can follow these commands from a root project directory.
+
+```
+dotnet restore
+dotnet build --configuration Release --output "./release" --no-restore
+dotnet release\OpenFeature.Benchmarks.dll

```

Comment:
```suggestion
dotnet release/OpenFeature.Benchmarks.dll
```

I think this should work on powershell, right?

Reply:
thanks for the comment. updated